app/views/members/_form.html.haml

```diff
@@ -45,6 +45,12 @@
         .small-10.large-10.column
           = f.input :email, placeholder: 'Email Address', label: false
 
+      .row.collapse.prefix-radius
+        .column.small-2.large-2
+          %span.prefix Phone
+        .small-10.large-10.column
+          = f.input :phone, placeholder: 'Phone Number', label: false
```

Comment:
Let's have the value displayed as `format_phone` for increased legibility, rather than just the numbers. Check out the `input_html` option for Simple Form [here](https://github.com/plataformatec/simple_form#usage)